schemas/aws/account-1.yml

```diff
@@ -43,6 +43,24 @@ properties:
       required:
       - name
       - email
+  securityContact:
+    type: object
+    properties:
+      name:
+        type: string
+        description: "The name of the security contact."
+      title:
+        type: string
+        description: "The title of the security contact. Default use the name."
+      email:
+        type: string
+        format: email
+      phoneNumber:
+        type: string
```

Comment:
should we enforce something here, like `^+\d+` with a max length of 16? Just to make sure that we use proper international numbers here...

Reply:
or at least what AWS accepts: https://docs.aws.amazon.com/accounts/latest/reference/manage-acct-update-contact-alternate.html#manage-acct-update-contact-alternate-requirements

Reply:
I added a very basic and simple phone number check. Should be enough for our use-case.